fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -2315,6 +2324,11 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueue* self,
 							    .detail("JobID", rd.bulkLoadTask.get().coreState.getJobId())
 							    .detail("TaskID", rd.bulkLoadTask.get().coreState.getTaskId());
 						}
+						for (const auto& [team, _] : bestTeams) {
+							for (const UID& ssid : team->getServerIDs()) {
+								self->bulkLoadTaskCollection->busyMap.removeTask(ssid);
```

Comment:
do we need to do the same in the above `throw bulkLoadError;`?

Reply:
Nice catch!